core/config.go

```diff
@@ -261,6 +263,8 @@ type Config struct {
 	// - CLI
 	// - Frontend
 	Components []string `yaml:"components,omitempty"`
+	// Text editor for editing texts. eg. body text
+	TextEditorName string `yaml:"textEditorName,omitempty" default:"vim"`
```

Comment:
We don't want any config for your editor as that is a personal config where the config file is for project configuration. Instead you should read from the environment via `os.Getenv`.

Reply:
ok.